.github/workflows/_validate.yaml

```diff
@@ -0,0 +1,35 @@
+name: "Code Review"
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  ci:
+    if: ${{ github.actor != 'dx-pagopa-bot' }}
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: "read"
+      actions: "read"
+      pull-requests: "read"
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@93cb6efe18208431cddfb8368fd83d5badbf9bfd # v5.0.1
+        with:
+          fetch-depth: 0
+          filter: tree:0
+
+      - name: Install Node.js
+        uses: pagopa/dx/.github/actions/node-setup@main
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Sets the base and head SHAs required for `nx affected` command
+        uses: nrwl/nx-set-shas@3e9ad7370203c1e93d109be57f3b72eb0eb511b1
+
+      - name: Run code quality checks on affected projects
+        run: nx affected --target=build,test,lint,format:check --configuration=ci
```

Comment:
thoughts: why not a single composite `validate` task? so you can configure this outside the GHA and call a reusable Workflow